integration-cli/docker_cli_attach_unix_test.go

```diff
@@ -80,7 +79,11 @@ func (s *DockerSuite) TestAttachAfterDetach(c *check.C) {
 
 	select {
 	case err := <-errChan:
-		c.Assert(err, check.IsNil)
+		if err != nil {
+			buff := make([]byte, 200)
+			tty.Read(buff)
+			c.Fatalf("%s: %s", err, buff)
+		}
```

Comment:
This was just to improve the error message as I was debugging the test failure. I can remove it, but I thought I'd leave it in case someone else hits a failure here.